src/main/kotlin/no/skatteetaten/aurora/boober/controller/v2/MultiAffiliationControllerV2.kt

```diff
@@ -0,0 +1,79 @@
+package no.skatteetaten.aurora.boober.controller.v2
+
+import com.fasterxml.jackson.annotation.JsonInclude
+import mu.KotlinLogging
+import no.skatteetaten.aurora.boober.controller.internal.ErrorsResponse
+import no.skatteetaten.aurora.boober.controller.v1.getRefNameFromRequest
+import no.skatteetaten.aurora.boober.facade.AuroraConfigFacade
+import no.skatteetaten.aurora.boober.feature.applicationDeploymentRef
+import no.skatteetaten.aurora.boober.feature.cluster
+import no.skatteetaten.aurora.boober.feature.envName
+import no.skatteetaten.aurora.boober.model.ApplicationDeploymentRef
+import no.skatteetaten.aurora.boober.model.toAdr
+import no.skatteetaten.aurora.boober.service.AuroraConfigRef
+import no.skatteetaten.aurora.boober.utils.parallelMap
+import org.springframework.web.bind.annotation.GetMapping
+import org.springframework.web.bind.annotation.PathVariable
+import org.springframework.web.bind.annotation.RequestMapping
+import org.springframework.web.bind.annotation.RestController
+
+private val logger = KotlinLogging.logger {}
+
+@RestController
+@RequestMapping("/v2/multiaffiliation")
+class MultiAffiliationControllerV2(
+    private val auroraConfigFacade: AuroraConfigFacade
+) {
+
+    @GetMapping("/{environment}")
+    fun findAll(@PathVariable environment: String): ErrorsResponse {
+        val refName = getRefNameFromRequest()
+        // val refName = "feature/AOS-5477_hente_miljo_paa_tvers"
+
+        val allApplications: List<MultiAffiliationResponse> =
```

Comment:
Etter min mening så er det litt for mye logikk i controlleren. Det er mye her som hører hjemme i en service eller facade. Med tanke på forandringen med search så kunne man kanskje sett for seg at man skal ha en ApplicationSearchService og der kunne man nok hatt mye av dette. Man måtte da hatt en dto som går fra service laget til controlleren også gjør controlleren det om til en response.

Reply:
Et annet alternativ er å legge det i auroraConfigFacade for nå fram til man får mer realtert til search. Men da er det viktig at "MultiAffiliationResponse" blir laget i controller og ikke i facade